package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Todo",
+  "version": "1.0.0",
+  "description": "a JavaScript project",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
Let's add the following scripts so we can initialize the webpack and build our project

```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```